src/tests/Builder.test.js

```diff
@@ -0,0 +1,594 @@
+         expect(builder.isExcludedResource(tbl1idW)).to.be(false);
+         expect(builder.isExcludedResource(tbl2R)).to.be(false);
+         expect(builder.isExcludedResource(tbl2W)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idW)).to.be(false);
+
+         builder.excludeTable(tbl1W.tableName, tbl1W.capacityType);
+         expect(builder.isExcludedResource(tbl1R)).to.be(true);
+         expect(builder.isExcludedResource(tbl1W)).to.be(true);
+         expect(builder.isExcludedResource(tbl1idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl1idW)).to.be(false);
+         expect(builder.isExcludedResource(tbl2R)).to.be(false);
+         expect(builder.isExcludedResource(tbl2W)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idW)).to.be(false);
+      });
+
+      it('excludes a particular resource for capacity type - indexes', function() {
+         builder.excludeIndex(tbl1idR.tableName, tbl1idR.indexName, tbl1idR.capacityType);
+         expect(builder.isExcludedResource(tbl1R)).to.be(false);
+         expect(builder.isExcludedResource(tbl1W)).to.be(false);
+         expect(builder.isExcludedResource(tbl1idR)).to.be(true);
+         expect(builder.isExcludedResource(tbl1idW)).to.be(false);
+         expect(builder.isExcludedResource(tbl2R)).to.be(false);
+         expect(builder.isExcludedResource(tbl2W)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idW)).to.be(false);
+
+         builder.excludeIndex(tbl1idW.tableName, tbl1idW.indexName, tbl1idW.capacityType);
+         expect(builder.isExcludedResource(tbl1R)).to.be(false);
+         expect(builder.isExcludedResource(tbl1W)).to.be(false);
+         expect(builder.isExcludedResource(tbl1idR)).to.be(true);
+         expect(builder.isExcludedResource(tbl1idW)).to.be(true);
+         expect(builder.isExcludedResource(tbl2R)).to.be(false);
+         expect(builder.isExcludedResource(tbl2W)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idW)).to.be(false);
+      });
+
+      it('excludes multiple resource for capacity type with wildcard - tables', function() {
+         builder.excludeTable('Tbl1*', constants.READ);
+         expect(builder.isExcludedResource(tbl1R)).to.be(true);
+         expect(builder.isExcludedResource(tbl1W)).to.be(false);
+         expect(builder.isExcludedResource(tbl1idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl1idW)).to.be(false);
+         expect(builder.isExcludedResource(tbl2R)).to.be(false);
+         expect(builder.isExcludedResource(tbl2W)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idW)).to.be(false);
```

Comment:
Same here ... I'll stop commenting on these... you can either change the `it` description to be more ambiguous (e.g. not say "excludes multiple resources") so that it's a correct statement for both tests it contains, or split the tests into multiple tests - your choice. It just seems odd to say "excludes multiple resources" and then immediately do a test that excludes only one.